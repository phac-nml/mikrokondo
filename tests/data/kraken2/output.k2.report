100.00	38	0	R	1	root
100.00	38	0	D	10239	  Viruses
 92.11	35	0	D1	2559587	    Riboviria
 84.21	32	0	P	2497569	      Negarnaviricota
 84.21	32	0	P1	2497571	        Polyploviricotina
 84.21	32	0	C	2497577	          Insthoviricetes
 84.21	32	0	O	2499411	            Articulavirales
 84.21	32	0	F	11308	              Orthomyxoviridae
 63.16	24	0	G	197911	                Alphainfluenzavirus
 63.16	24	0	S	11320	                  Influenza A virus
 21.05	8	0	S1	114727	                    H1N1 subtype
 21.05	8	8	S2	211044	                      Influenza A virus (A/Puerto Rico/8/1934(H1N1))
 21.05	8	0	S1	114729	                    H2N2 subtype
 21.05	8	8	S2	488241	                      Influenza A virus (A/Korea/426/1968(H2N2))
 21.05	8	0	S1	119210	                    H3N2 subtype
 21.05	8	8	S2	335341	                      Influenza A virus (A/New York/392/2004(H3N2))
 21.05	8	0	G	197912	                Betainfluenzavirus
 21.05	8	0	S	11520	                  Influenza B virus
 21.05	8	8	S1	518987	                    Influenza B virus (B/Lee/1940)
  7.89	3	0	O	76804	      Nidovirales
  7.89	3	0	O1	2499399	        Cornidovirineae
  7.89	3	0	F	11118	          Coronaviridae
  7.89	3	0	F1	2501931	            Orthocoronavirinae
  7.89	3	0	G	694002	              Betacoronavirus
  5.26	2	0	G1	2509511	                Sarbecovirus
  5.26	2	0	S	694009	                  Severe acute respiratory syndrome-related coronavirus
  5.26	2	2	S1	2697049	                    Severe acute respiratory syndrome coronavirus 2
  2.63	1	0	G1	2509494	                Merbecovirus
  2.63	1	1	S	1335626	                  Middle East respiratory syndrome-related coronavirus
  5.26	2	0	O	2169561	    Ortervirales
  5.26	2	0	F	11632	      Retroviridae
  5.26	2	0	F1	327045	        Orthoretrovirinae
  5.26	2	0	G	11646	          Lentivirus
  2.63	1	1	S	11676	            Human immunodeficiency virus 1
  2.63	1	1	S	11709	            Human immunodeficiency virus 2
  2.63	1	0	O	28883	    Caudovirales
  2.63	1	0	F	10699	      Siphoviridae
  2.63	1	0	G	186765	        Lambdavirus
  2.63	1	1	S	10710	          Escherichia virus Lambda
