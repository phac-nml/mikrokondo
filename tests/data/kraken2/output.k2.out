C	kraken:taxid|2697049|NC_045512.2	2697049	29903	2697049:40 694009:17 2697049:120 694009:7 2697049:30 694009:5 2697049:10 694009:16 2697049:42 694009:3 2697049:397 694009:5 2697049:5712 694009:7 2697049:2343 694009:4 2697049:1709 694009:5 2697049:449 694009:5 2697049:118 0:37 2697049:350 694009:1 2697049:1 694009:3 2697049:822 694009:2 2697049:60 694009:3 2697049:560 694009:11 2697049:1 694009:1 2697049:6 694009:4 2697049:234 694009:1 2697049:172 694009:1 2697049:32 694009:5 2697049:8 694009:5 2697049:15 694009:2 2697049:12 694009:2 2697049:60 694009:43 2697049:9 694009:1 2697049:263 694009:1 2697049:107 694009:1 2697049:409 694009:2 2697049:96 694009:5 2697049:12 694009:5 2697049:87 694009:5 2697049:145 694009:10 2697049:2 694009:3 2697049:45 694009:5 2697049:22 694009:11 2697049:78 694009:4 2697049:35 694009:2 2697049:7 694009:11 2697049:3 694009:1 2697049:51 694009:5 2697049:10 694009:6 2697049:6 694009:5 2697049:3 694009:1 2697049:17 694009:3 2697049:7 694009:3 2697049:31 694009:79 2697049:25 694009:2 2697049:20 694009:5 2697049:6 694009:4 2697049:264 694009:4 2697049:51 694009:4 2697049:321 694009:3 2697049:5 694009:12 2697049:1 694009:4 2697049:139 694009:5 2697049:41 694009:5 2697049:1 694009:5 2697049:11 694009:2 2697049:8 694009:1 2697049:121 694009:8 2697049:9 694009:5 2697049:565 694009:2 2697049:49 694009:13 2697049:3 694009:5 2697049:158 694009:1 2697049:7 694009:10 2697049:46 694009:5 2697049:8 694009:5 2697049:112 694009:4 2697049:53 694009:9 2697049:253 694009:3 2697049:35 694009:1 2697049:7 694009:9 2697049:282 694009:5 2697049:1 694009:5 2697049:92 694009:5 2697049:142 694009:1 2697049:410 694009:4 2697049:1039 694009:1 2697049:5 694009:2 2697049:200 694009:5 2697049:9 694009:5 2697049:325 694009:1 2697049:7 694009:1 2697049:606 694009:5 2697049:136 694009:1 2697049:626 694009:2 2697049:1325 694009:5 2697049:1124 694009:8 2697049:495 694009:2 2697049:166 694009:2 2697049:7 694009:2 2697049:35 694009:5 2697049:426 694009:2 2697049:269 694009:14 2697049:118 694009:3 2697049:684 694009:3 2697049:3 694009:8 2697049:34 694009:9 2697049:1 694009:1 2697049:5 694009:5 2697049:15 694009:32 2697049:16 694009:2 2697049:9 694009:35 2697049:7 694009:5 2697049:73 694009:5 2697049:115 694009:3 2697049:5 694009:2 2697049:129 694009:5 2697049:311 694009:5 2697049:9 694009:2 2697049:7 694009:1 2697049:116 694009:5 2697049:11 694009:5 2697049:2 694009:1 2697049:449 694009:5 2697049:129 694009:23 2697049:605 694009:7 2697049:5 694009:3 2697049:7 694009:1 2697049:87 694009:9 2697049:3 694009:4 2697049:5 694009:4 2697049:124 694009:3 2697049:3 694009:4 2697049:5 694009:3 2697049:68 694009:24 2697049:185 694009:5 2697049:21 694009:26 2697049:8 694009:1 2697049:87 694009:2 2697049:2 694009:1 2697049:39 694009:23 2697049:63 694009:2 2697049:94 694009:1 2697049:16 694009:5 2697049:234 694009:5 2697049:3 694009:1 2697049:1 694009:1 2697049:7 694009:5 2697049:47 694009:8 2697049:1 694009:9 2697049:21 694009:16 2697049:2 694009:7 2697049:19 694009:5 2697049:30 694009:69 0:33
C	kraken:taxid|211044|NC_002023.1	211044	2341	211044:4 11320:14 211044:9 11320:1 211044:129 11320:12 211044:3 11320:9 211044:172 11320:5 211044:47 11320:13 211044:123 11320:6 211044:54 11320:3 211044:5 11320:4 211044:62 11320:7 211044:5 11320:2 211044:139 11320:5 211044:11 11320:1 211044:7 11320:12 211044:102 11320:1 211044:3 11320:1 211044:174 11320:1 211044:1 11320:4 211044:162 11320:1 211044:18 11320:2 211044:6 11320:2 211044:6 11320:5 211044:189 11320:5 211044:7 11320:5 211044:9 11320:3 211044:5 11320:3 211044:16 11320:21 211044:10 11320:3 211044:43 11320:2 211044:6 11320:5 211044:1 11320:5 211044:1 11320:7 211044:26 11320:1 211044:47 11320:1 211044:7 11320:5 211044:100 11320:5 211044:13 11320:2 211044:28 11320:45 211044:44 11320:5 211044:31 11320:1 211044:1 11320:1 211044:1 11320:5 211044:119 11320:1 211044:11 11320:76 211044:6 11320:5 211044:10 11320:6 211044:5
C	kraken:taxid|211044|NC_002021.1	211044	2341	211044:4 11320:17 211044:1 11320:4 211044:77 11320:3 211044:5 11320:5 211044:931 11320:1 211044:176 11320:7 211044:4 11320:5 211044:298 11320:2 211044:701 11320:61 211044:5
C	kraken:taxid|211044|NC_002022.1	211044	2233	211044:8 11320:5 211044:1 11320:1 211044:6 11320:15 211044:3 11320:3 211044:5 11320:2 211044:54 11320:40 211044:37 11320:7 211044:44 11320:1 211044:3 11320:5 211044:8 11320:2 211044:5 11320:33 211044:5 11320:4 211044:26 11320:8 211044:21 11320:5 211044:9 11320:5 211044:45 11320:5 211044:1 11320:5 211044:28 11320:1 211044:3 11320:5 211044:7 11320:13 211044:7 11320:5 211044:1 11320:5 211044:5 11320:2 211044:15 11320:5 211044:24 11320:2 211044:5 11320:2 211044:11 11320:7 211044:7 11320:5 211044:52 11320:5 211044:9 11320:12 211044:19 11320:3 211044:5 11320:5 211044:2 11320:26 211044:13 11320:5 211044:105 11320:7 211044:7 11320:3 211044:17 11320:1 211044:5 11320:1 211044:41 11320:7 211044:6 11320:5 211044:35 11320:6 211044:5 11320:5 211044:1 11320:5 211044:21 11320:5 211044:13 11320:5 211044:16 0:31 488241:3 0:4 211044:5 11320:8 211044:133 11320:1 211044:60 11320:13 211044:3 11320:1 211044:144 11320:1 211044:46 11320:5 211044:3 11320:1 211044:1 11320:5 211044:11 11320:38 211044:39 11320:3 211044:20 11320:2 211044:55 11320:3 211044:3 11320:4 211044:28 11320:5 211044:65 11320:8 211044:5 11320:7 211044:77 11320:6 211044:8 11320:4 211044:56 11320:8 211044:59 11320:1 211044:2 11320:1 211044:5 11320:20 211044:21 11320:1 211044:21 11320:1 211044:41 11320:5 211044:10 11320:18
C	kraken:taxid|211044|NC_002017.1	211044	1778	211044:1247 0:65 211044:178 11320:5 211044:249
C	kraken:taxid|211044|NC_002019.1	211044	1565	211044:25 11320:19 211044:5 11320:5 211044:87 11320:8 211044:11 11320:13 211044:46 11320:16 211044:58 11320:14 211044:61 11320:15 211044:7 11320:1 211044:40 11320:3 211044:6 11320:8 211044:49 11320:12 211044:4 11320:5 211044:19 11320:5 211044:67 11320:3 211044:1 11320:5 211044:7 11320:13 211044:1 11320:5 211044:3 11320:5 211044:164 11320:5 211044:55 11320:5 211044:146 11320:6 211044:8 11320:2 211044:7 11320:4 211044:134 11320:18 211044:110 11320:3 211044:89 11320:22 211044:3 11320:2 211044:42 11320:31 211044:23
C	kraken:taxid|211044|NC_002018.1	211044	1413	211044:1379
C	kraken:taxid|211044|NC_002016.1	211044	1027	211044:2 11320:10 211044:13 11320:1 211044:2 11320:13 211044:4 11320:2 211044:17 11320:25 211044:9 11320:7 211044:32 11320:1 211044:5 11320:27 211044:37 11320:36 211044:6 11320:1 211044:33 11320:16 211044:17 11320:1 211044:5 11320:1 211044:9 11320:3 211044:36 11320:3 211044:2 11320:1 211044:28 11320:25 211044:1 11320:5 211044:2 11320:2 211044:7 11320:5 211044:7 11320:13 211044:7 11320:1 211044:86 11320:55 211044:5 11320:3 211044:1 11320:5 211044:42 11320:1 211044:2 11320:5 211044:17 11320:5 211044:7 11320:37 211044:37 11320:5 211044:5 11320:4 211044:2 11320:15 0:35 211044:65 11320:24 211044:2 11320:4 211044:15 11320:5 211044:1 11320:2 211044:6 11320:15
C	kraken:taxid|211044|NC_002020.1	211044	890	211044:3 11320:5 211044:25 11320:4 211044:56 11320:21 211044:7 11320:5 211044:98 11320:1 211044:7 11320:10 211044:222 11320:41 211044:57 11320:18 211044:6 11320:5 211044:5 11320:5 211044:5 11320:1 211044:5 11320:39 211044:111 11320:21 211044:46 11320:11 211044:16
C	kraken:taxid|488241|NC_007378.1	488241	2341	488241:2 11320:16 488241:9 11320:6 488241:54 11320:17 488241:3 11320:9 488241:11 11320:1 488241:3 11320:50 488241:26 11320:5 488241:29 11320:7 488241:105 11320:5 488241:21 11320:13 488241:13 11320:13 488241:44 11320:7 488241:1 11320:5 488241:94 11320:5 488241:10 11320:5 488241:12 11320:2 488241:4 11320:6 488241:2 11320:2 488241:6 11320:21 488241:4 11320:5 488241:1 11320:5 488241:16 11320:11 488241:1 11320:1 488241:62 11320:5 488241:82 11320:27 488241:20 11320:4 488241:45 11320:8 488241:5 11320:1 488241:12 11320:2 488241:5 11320:1 488241:3 11320:1 488241:75 11320:2 488241:74 11320:1 488241:7 11320:5 488241:5 11320:6 488241:1 11320:4 488241:28 11320:10 488241:41 11320:5 488241:9 11320:13 488241:56 11320:1 488241:50 11320:5 488241:44 11320:5 488241:18 11320:2 488241:5 11320:1 488241:98 11320:5 488241:7 11320:5 488241:9 11320:3 488241:5 11320:3 488241:16 11320:21 488241:8 11320:5 488241:43 11320:3 488241:5 11320:5 488241:1 11320:5 488241:1 11320:7 488241:27 11320:1 488241:46 11320:1 488241:7 11320:5 488241:54 11320:3 488241:1 11320:6 488241:54 11320:2 488241:29 11320:49 488241:32 11320:2 488241:5 11320:5 488241:31 11320:1 488241:1 11320:1 488241:1 11320:5 488241:67 11320:1 488241:9 11320:5 488241:49 11320:108
C	kraken:taxid|488241|NC_007375.1	488241	2341	11320:21 488241:1 11320:4 488241:57 11320:13 488241:7 11320:5 488241:3 11320:5 488241:7 11320:11 488241:9 11320:5 488241:74 11320:1 488241:277 11320:4 488241:1 11320:2 488241:6 11320:2 488241:23 11320:3 488241:49 11320:9 488241:2 11320:3 488241:92 11320:3 488241:9 11320:1 488241:160 11320:2 488241:3 11320:1 488241:310 11320:1 488241:1 11320:5 488241:33 11320:3 488241:6 11320:6 488241:51 A:34 488241:43 11320:9 488241:5 11320:5 488241:233 11320:1 488241:10 11320:5 488241:281 11320:5 488241:238 11320:1 488241:80 11320:66
C	kraken:taxid|488241|NC_007376.1	488241	2233	11320:36 488241:3 11320:3 488241:5 11320:2 488241:54 11320:45 488241:10 11320:5 488241:17 11320:7 488241:44 11320:1 488241:3 11320:5 488241:8 11320:2 488241:5 11320:33 488241:5 11320:4 488241:35 11320:5 488241:4 11320:11 488241:7 11320:5 488241:50 11320:7 488241:1 11320:5 488241:28 11320:1 488241:3 11320:5 488241:4 11320:20 488241:3 11320:5 488241:1 11320:5 488241:5 11320:2 488241:15 11320:5 488241:19 11320:56 488241:54 11320:12 488241:7 11320:2 488241:10 11320:41 488241:8 11320:10 488241:105 11320:7 488241:7 11320:3 488241:17 11320:1 488241:5 11320:1 488241:41 11320:7 488241:5 11320:6 488241:31 11320:11 488241:1 11320:8 488241:1 11320:5 488241:2 11320:5 488241:14 11320:5 488241:13 11320:5 488241:39 11320:1 488241:19 11320:8 488241:19 11320:1 488241:19 11320:3 488241:12 11320:39 488241:13 11320:5 488241:22 11320:1 488241:60 11320:9 488241:7 11320:1 488241:106 11320:1 488241:5 11320:1 488241:5 11320:1 488241:21 11320:5 488241:15 11320:2 488241:29 11320:5 488241:3 11320:1 488241:1 11320:5 488241:13 11320:45 488241:4 11320:4 488241:22 11320:3 488241:14 11320:4 488241:2 11320:2 488241:38 11320:5 488241:1 11320:6 488241:5 11320:1 488241:5 11320:4 488241:28 11320:5 488241:65 11320:12 488241:1 11320:18 488241:31 11320:6 488241:2 11320:2 488241:5 11320:2 488241:8 11320:3 488241:7 11320:6 488241:8 11320:4 488241:56 11320:8 488241:68 11320:37 488241:4 11320:1 488241:63 11320:5 488241:10 11320:18
C	kraken:taxid|488241|NC_007374.1	488241	1773	488241:1489 11320:1 488241:249
C	kraken:taxid|488241|NC_007381.1	488241	1497	11320:22 488241:4 11320:5 488241:65 11320:8 488241:11 11320:13 488241:46 11320:16 488241:58 11320:14 488241:32 11320:6 488241:6 11320:6 488241:7 11320:1 488241:5 11320:13 488241:7 11320:3 488241:38 11320:3 488241:6 11320:7 488241:4 11320:5 488241:41 11320:12 488241:4 11320:5 488241:19 11320:5 488241:67 11320:3 488241:1 11320:5 488241:7 11320:13 488241:1 11320:5 488241:3 11320:5 488241:55 11320:1 488241:14 11320:7 488241:29 11320:5 488241:53 11320:5 488241:10 11320:5 488241:40 11320:5 488241:114 11320:22 488241:4 11320:5 488241:15 11320:1 488241:11 11320:1 488241:132 11320:20 488241:9 11320:5 488241:3 11320:5 488241:86 11320:1 488241:1 11320:3 488241:89 11320:22 488241:3 11320:2 488241:16 11320:11 488241:5 11320:5 488241:7 11320:29
C	kraken:taxid|488241|NC_007382.1	488241	1410	488241:3 11320:1 488241:243 11320:7 488241:11 11320:2 488241:2 11320:1 488241:6 11320:10 488241:238 11320:10 488241:7 11320:2 488241:42 11320:1 488241:98 11320:2 488241:24 11320:4 488241:144 11320:5 488241:59 11320:5 488241:1 11320:5 488241:311 11320:5 488241:53 11320:5 488241:1 11320:6 488241:41 11320:6 488241:15
C	kraken:taxid|488241|NC_007377.1	488241	1027	11320:20 488241:5 11320:16 488241:4 11320:2 488241:17 11320:27 488241:7 11320:7 488241:5 11320:1 488241:1 11320:9 488241:7 11320:42 488241:26 11320:5 488241:17 11320:6 488241:5 11320:27 488241:11 11320:5 488241:11 11320:7 488241:26 11320:1 488241:5 11320:1 488241:9 11320:5 488241:34 11320:21 488241:6 11320:23 488241:37 11320:14 488241:5 11320:3 488241:84 11320:57 488241:5 11320:3 488241:1 11320:15 488241:7 11320:1 488241:22 11320:2 488241:25 11320:9 488241:3 11320:42 488241:63 0:35 488241:43 11320:1 488241:1 11320:44 488241:5 11320:1 488241:10 11320:1 488241:4 11320:5 488241:1 11320:2 488241:2 11320:19
C	kraken:taxid|488241|NC_007380.1	488241	838	488241:9 11320:2 488241:56 11320:22 488241:6 11320:10 488241:93 11320:1 488241:7 11320:11 488241:33 11320:5 488241:6 11320:2 488241:37 11320:6 488241:12 11320:1 488241:20 11320:2 488241:35 11320:12 488241:4 11320:2 488241:35 11320:50 488241:2 11320:2 488241:12 11320:61 488241:4 11320:5 488241:5 11320:5 488241:5 11320:47 488241:6 11320:1 488241:6 11320:5 488241:35 11320:2 488241:5 11320:10 488241:39 11320:21 488241:30 11320:5 488241:12
C	kraken:taxid|335341|NC_007373.1	335341	2341	335341:28 11320:5 335341:54 11320:17 335341:3 11320:9 335341:11 11320:1 335341:3 11320:26 335341:3 11320:5 335341:42 11320:1 335341:33 11320:7 335341:131 11320:13 335341:70 11320:7 335341:1 11320:5 335341:66 11320:6 335341:23 11320:4 335341:10 11320:5 335341:26 11320:2 335341:6 11320:21 335341:4 11320:5 335341:1 11320:5 335341:91 11320:1 335341:77 11320:5 335341:4 11320:22 335341:25 11320:4 335341:45 11320:9 335341:4 11320:1 335341:12 11320:2 335341:82 11320:5 335341:74 11320:1 335341:7 11320:5 335341:5 11320:5 335341:31 11320:13 335341:41 11320:5 335341:9 11320:13 335341:75 11320:2 335341:6 11320:2 335341:6 11320:5 335341:11 11320:5 335341:44 11320:5 335341:18 11320:2 335341:5 11320:1 335341:396 11320:3 335341:1 11320:6 335341:36 11320:5 335341:54 11320:2 335341:10 11320:27 335341:29 11320:5 335341:117 11320:1 335341:9 11320:5 335341:34 11320:4 335341:13 11320:106
C	kraken:taxid|335341|NC_007372.1	335341	2341	11320:8 335341:75 11320:13 335341:26 11320:12 335341:9 11320:5 335341:73 11320:2 335341:277 11320:4 335341:1 11320:2 335341:6 11320:2 335341:23 11320:3 335341:49 11320:9 335341:2 11320:3 335341:92 11320:1 335341:8 11320:5 335341:159 11320:2 335341:3 11320:1 335341:172 11320:1 335341:137 11320:1 335341:1 11320:5 335341:28 11320:8 335341:6 11320:1 335341:133 11320:9 335341:5 11320:5 335341:86 0:15 488241:1 0:23 335341:26 11320:2 335341:80 11320:1 335341:10 11320:5 335341:281 11320:5 335341:238 11320:1 335341:80 11320:4 335341:11 11320:3 335341:35 11320:13
C	kraken:taxid|335341|NC_007371.1	335341	2233	11320:22 335341:4 11320:5 335341:77 11320:1 335341:1 11320:5 335341:4 11320:1 335341:23 11320:5 335341:10 11320:5 335341:113 11320:1 335341:1 11320:5 335341:40 11320:14 335341:4 11320:7 335341:3 11320:6 335341:2 11320:5 335341:2 11320:5 335341:43 11320:1 335341:1 11320:5 335341:1 11320:5 335341:125 11320:52 335341:40 11320:5 335341:28 11320:2 335341:10 11320:24 335341:25 11320:5 335341:3 11320:3 335341:198 11320:1 335341:44 11320:3 335341:1 11320:3 335341:13 11320:5 335341:76 11320:1 335341:42 11320:5 335341:19 11320:3 335341:12 11320:39 335341:13 11320:5 335341:206 11320:1 335341:5 11320:1 335341:5 11320:1 335341:41 11320:2 335341:74 11320:28 335341:4 11320:4 335341:39 11320:4 335341:42 11320:12 335341:5 11320:2 335341:36 11320:5 335341:78 11320:18 335341:31 11320:6 335341:2 11320:2 335341:5 11320:2 335341:9 11320:2 335341:145 11320:4 335341:2 11320:1 335341:5 11320:5 335341:6 11320:23 335341:29 11320:1 335341:66 11320:3 335341:3 11320:1 335341:1
C	kraken:taxid|335341|NC_007366.1	335341	1762	335341:1728
C	kraken:taxid|335341|NC_007369.1	335341	1566	335341:25 11320:40 335341:6 11320:5 335341:84 11320:7 335341:172 11320:6 335341:6 11320:6 335341:7 11320:1 335341:82 11320:9 335341:251 11320:1 335341:14 11320:7 335341:29 11320:5 335341:68 11320:5 335341:159 11320:22 335341:4 11320:5 335341:1 11320:6 335341:159 11320:5 335341:21 11320:2 335341:3 11320:5 335341:86 11320:1 335341:1 11320:3 335341:132 11320:11 335341:5 11320:5 335341:7 11320:3 335341:1 11320:11 335341:38
C	kraken:taxid|335341|NC_007368.1	335341	1467	335341:22 11320:1 335341:92 0:38 335341:111 11320:9 335341:11 11320:2 335341:2 11320:1 335341:6 11320:10 335341:238 11320:10 335341:7 11320:2 335341:42 11320:1 335341:41 0:37 335341:18 11320:4 335341:24 11320:4 335341:144 11320:5 335341:61 11320:3 335341:1 11320:5 335341:311 11320:5 335341:55 11320:3 335341:1 11320:6 335341:41 11320:6 335341:34 0:19
C	kraken:taxid|335341|NC_007367.1	335341	1027	11320:21 335341:51 11320:5 335341:13 11320:1 335341:7 11320:7 335341:5 11320:1 335341:1 11320:9 335341:6 11320:16 335341:53 11320:1 335341:7 11320:52 335341:11 11320:5 335341:13 11320:14 335341:72 11320:21 335341:6 11320:7 335341:19 11320:6 335341:1 11320:5 335341:3 11320:1 335341:7 11320:5 335341:25 11320:2 335341:87 11320:15 335341:1 11320:5 335341:48 11320:14 335341:3 11320:1 335341:27 11320:5 335341:17 11320:5 335341:7 11320:7 335341:13 11320:1 335341:3 11320:16 335341:34 11320:5 335341:5 11320:4 335341:2 11320:15 0:35 335341:43 11320:1 335341:1 11320:25 335341:8 11320:5 335341:22 11320:1 335341:21 11320:7 335341:5
C	kraken:taxid|335341|NC_007370.1	335341	890	335341:3 11320:5 335341:99 11320:8 335341:11 11320:5 335341:145 11320:5 335341:6 11320:2 335341:37 11320:6 335341:12 11320:1 335341:20 11320:2 335341:35 11320:9 335341:7 11320:2 335341:35 11320:19 335341:8 11320:5 335341:20 11320:2 335341:13 11320:60 335341:4 11320:5 335341:15 11320:5 335341:3 11320:10 335341:1 11320:3 335341:31 11320:1 335341:8 11320:3 335341:35 11320:2 335341:5 11320:10 335341:90 11320:5 335341:11 11320:11 335341:16
C	kraken:taxid|518987|NC_002204.1	518987	2368	518987:2334
C	kraken:taxid|518987|NC_002205.1	518987	2313	518987:530 0:48 518987:516 0:37 518987:1148
C	kraken:taxid|518987|NC_002206.1	518987	2204	518987:2170
C	kraken:taxid|518987|NC_002207.1	518987	1882	518987:1848
C	kraken:taxid|518987|NC_002208.1	518987	1841	518987:1807
C	kraken:taxid|518987|NC_002209.1	518987	1557	518987:1523
C	kraken:taxid|518987|NC_002210.1	518987	1191	518987:1157
C	kraken:taxid|518987|NC_002211.1	518987	1096	518987:1062
C	kraken:taxid|11676|NC_001802.1	11676	9181	11676:2360 0:46 11676:1903 0:45 11676:13 0:75 11676:1147 0:79 11676:593 0:37 11676:1638 0:65 11676:1146
C	kraken:taxid|11709|NC_001722.1	11709	10359	11709:2983 0:79 11709:5743 0:39 11709:1481
C	kraken:taxid|10710|NC_001416.1	10710	48502	10710:2395 0:40 10710:3646 0:50 10710:4490 0:36 10710:138 0:40 10710:11500 0:36 10710:390 0:38 10710:927 0:44 10710:1076 0:39 10710:1806 0:38 10710:155 0:38 10710:3905 0:40 10710:4761 0:55 10710:2144 0:42 10710:256 0:40 10710:24 0:38 10710:879 0:46 10710:7557 0:39 10710:1720
C	kraken:taxid|1335626|NC_019843.3	1335626	30119	1335626:2811 0:40 1335626:27226 0:8
C	kraken:taxid|694009|NC_004718.3	2697049	29751	694009:22482 0:39 694009:13 0:41 694009:6619 2697049:16 0:2 2697049:5 0:25 694009:451 0:24
